core/order.go

```diff
@@ -514,11 +513,7 @@ func (n *OpenBazaarNode) createContractWithOrder(data *PurchaseData) (*pb.Ricard
 			if err := validateVendorID(sl.Listing); err != nil {
 				return nil, err
 			}
-			testnet := false
-			if n.Wallet.Params().Name != chaincfg.MainNetParams.Name {
-				testnet = true
-			}
-			if err := validateListing(sl.Listing, testnet); err != nil {
+			if err := validateListing(sl.Listing, n.TestNetworkEnabled()); err != nil {
```

Comment:
This line needs to be `n.TestNetworkEnabled() || n.RegressionTestEnabled()` to make it pass the QA test.

Reply:
That does not appear to be enough to get QA to pass. Have you tried it?